input.go

```diff
@@ -52,6 +52,11 @@ func CursorPosition() (x, y int) {
 	return ui.AdjustedCursorPosition()
 }
 
+// MouseScroll returns the y offset of the mouse wheel
+func MouseScroll() float64 {
+	return input.Get().MouseScroll()
```

Comment:
You should implement this function not only at `input_glfw.go` but also `input_js.go` and `input_mobile.go`. I think it should be possible to implement this for browsers. For mobiles, just return 0 is fine.

Reply:
Ok, will do!

Reply:
I implemented the function at `input_mobile.go`  but failed to do so at `input_js.go`. Sry, I'm too inexperienced to see through that.